src/applications/hca/containers/IntroductionPage.jsx

```diff
@@ -62,7 +58,7 @@ const VerificationRequiredAlert = () => (
           </a>
         </li>
         <li>
-          Or call us at <a href="tel:+18772228387">877-222-8387</a>. If you have
+          Or call us at <a href="tel:+18772228387">877-222-8387</a>. If you have{' '}
```

Comment:
should we use the `<Telephone/> `component here also? not sure if theres a reason why its not used?

Reply:
Yeah probably, I will update that and then merge it in